client/Rating&Reviews/PhotoList.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PhotoList = ({photos}) => (
+  <div>
+    {photos.map(photo => <img key={photo.id} className='photo' src={photo.url} alt='product' />)}
+  </div>
+)
+
+PhotoList.propTypes = {
+  photos: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.number,
+    url: PropTypes.string
+  }))
+}
+
+PhotoList.defaultProps = {
+  photos: []
```

Comment:
If you do not specify `.isRequired` for a particular prop during validation, then the linter demands you assign a default for that prop like so.

Reply:
Nice use of defaultProps.